### PR TITLE
empty test case

### PR DESCRIPTION
**Pull request by tnishida1**

### CODE REVIEW

File: tests/lab01_test.go [hunk 1]
```diff
@@ -29,10 +29,14 @@ var _ = Describe("Lab 1", func() {
 		})
 
 		It("should have a valid skaffold.yaml", func() {
-			skaffoldExpected := ExpectYamlToParse("../skaffold.yaml")
+			skaffoldExpected, errorMessage := ExpectYamlToParse("../skaffold.yaml")
+			if errorMessage != "" {
+				failMessage = errorMessage
+			}
```
Comment (sjordan8) [inline tests/lab01_test.go:35]: Couldn't you just set failMessage directly on line 32?

File: tests/lab01_test.go [hunk 2]
```diff
@@ -71,10 +75,14 @@ var _ = Describe("Lab 1", func() {
 
 	Context("Step 11", func() {
 		It("skaffold file should have a profile section", func() {
-			skaffoldExpected := ExpectYamlToParse("../skaffold.yaml")
+			skaffoldExpected, errorMessage := ExpectYamlToParse("../skaffold.yaml")
+			if errorMessage != "" {
+				failMessage = errorMessage
+			}
```
Comment (sjordan8) [inline tests/lab01_test.go:81]: Same as above

File: tests/lab02_test.go [hunk 3]
```diff
@@ -21,10 +21,14 @@ var _ = Describe("Lab 2", func() {
 			Expect("../skaffold.yaml").To(BeAnExistingFile(), failMessage)
 		})
 		It("should have a valid skaffold.yaml", func() {
-			skaffoldExpected := ExpectYamlToParse("../skaffold.yaml")
+			skaffoldExpected, errorMessage := ExpectYamlToParse("../skaffold.yaml")
+			if errorMessage != "" {
+				failMessage = errorMessage
+			}
```
Comment (sjordan8) [inline tests/lab02_test.go:27]: Same as above

**Review by sjordan8 — CHANGES_REQUESTED** [review]

File: tests/lab01_test.go (hunk 1 above)
Comment (tnishida1) [reply tests/lab01_test.go:35]: So when I tried to directly pass the failMessage, it wouldn't pass the value of failMessage down to the AfterEach to concatenate the value into the `slack-output.md`. I didn't understand why exactly the value isn't passed but this was the workaround for that

**Review by sjordan8 — APPROVED** [review]

**Pull request merged**